ARMTemplates/AzFunctionApp-Consumption.JSON

```diff
@@ -0,0 +1,141 @@
+{
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+	  "appName": {
+		"type": "string",
+		"defaultValue": "[concat('fnapp', uniqueString(resourceGroup().id))]",
```

Comment:
Is this default value needed if this parameter is defined in the `parameters.json `file? Can the default value refer to `parameters.json`? Speaking of which, perhaps these two files should have names related to the project, rather than just basic Azure Function boilerplate?

Reply:
I agree! i just want the parameters file to be separate all together where they are defined and values can be set as well. although, i was get errors when adding `defaultValue ` & `allowedValues `to the parameters.json file which was weird :| I remember doing this sometime back so not sure if i am missing something silly. will try adn fix in the second iteration

Reply:
Maybe we can edit the `allowedValues` in the main json file, and just keep the `defaultValue` the same as the `defaultValue` in the parameters json. It'd be nice if it could pull that way, but it's okay if it can't.

Reply:
since we are not creating them as mandatory params, leaving a `defaultvalue` helps with a quick smoke test maybe?

Reply:
Yeah that sounds good to me.